packages/lesswrong/components/common/TabNavigationMenu/TabNavigationMenu.tsx

```diff
@@ -85,7 +85,7 @@ const TabNavigationMenu = ({onClickSection, transparentBackground, classes}: {
             />
           })}
           {/* NB: This returns null if you don't have any active resources */}
-          <FeaturedResourceBanner terms={{view: "activeResources"}}/>
+          {/* <FeaturedResourceBanner terms={{view: "activeResources"}}/> */}
```

Comment:
This doesn't play nicely with some screen sizes, and some of these ads are linked in the new RHS anyway, so we're hiding this for now.

![Screen Shot 2023-08-07 at 5 32 08 PM](https://github.com/ForumMagnum/ForumMagnum/assets/9057804/0e68333f-3d5f-49e9-a681-73360d6d7b58)